idaes/gas_solid_contactors/flowsheets/ss_MB_methane_combustion.py

```diff
@@ -112,14 +112,26 @@ def main():
             'Fe3O4': blk.solid_inlet.mass_frac_comp[0, 'Fe3O4'].value,
             'Al2O3': blk.solid_inlet.mass_frac_comp[0, 'Al2O3'].value}}
 
+    print()
+    print("Apply scaling transformation")
+    # Scale the model by applying scaling transformation
+    # This reduces ill conditioning of the model
+    iscale.calculate_scaling_factors(m)
+
+    print()
+    print("Initialize the model")
     m.fs.MB.initialize(outlvl=idaeslog.INFO,
+                       optarg={'tol': 1e-5},
                        gas_phase_state_args=gas_phase_state_args,
                        solid_phase_state_args=solid_phase_state_args)
 
     t_initialize = time.time()  # Initialization time
 
+    print()
+    print("Solve the model")
     # Create a solver
     solver = get_solver()
+    solver.options = {'tol': 1e-5}
```

Comment:
You need to be careful when doing this as this actually changes the global solver settings (i.e. it can effect other models running after this, such as during testing).

Reply:
There is a context manager somewhere to do this - the quickest way is probably to ask @eslickj.

Reply:
I actually think this is okay.  The line in question only sets options on that specific solver object.

Reply:
Thanks @eslickj